scripts/safe_template_linter.py

```diff
@@ -399,6 +549,8 @@ def _check_mako_file_is_safe(self, mako_template, results):
             results: A file results objects to which violations will be added.
 
         """
+        if self._is_django_template(mako_template):
+            return
```

Comment:
I think the platform has 1 file with both Django and Mako expressions.  I'm not sure what is going on with it, and if it is really processed by both, or some old file.
This code is really a hack because I just don't know the real Mako directories that get processed.  This should be addressed separately.